src/bitcoin-cli.cpp

```diff
@@ -307,6 +308,37 @@ bool SetStdinEcho(bool enable = true)
     return true;
 }
 
+tl::expected<UniValue, UniValue>
+CallRPC(const std::string &method, const std::vector<std::string> &strArgs)
+{
+    auto args =
```

Comment:
Similarly to above, we should keep the naming here consistent with the existing code:
```suggestion
CallRPC(const std::string &strMethod, const std::vector<std::string> &strParams)
{
    auto params =
```

Reply:
I made the params -> args changes because now this code deals with both the parameters and arguments, and it was very confusing to see arguments called parameters in that case. When I go back over this, I’ll either revert the renaming or come back with some more specific questions about how avoid the confusing bits.

Reply:
Erm, which are the parameters and which are the arguments? Definitely some naming clarification would be helpful. :sweat_smile:

Reply:
Arguments are the values passed to a function that is parameterized over some set of variables (the parameters).